addon/models/radar.js

```diff
@@ -275,14 +275,14 @@ export default class Radar {
 
     this._scrollHandler = () => {
       if (this.isTracking) {
-        this._nextScroll = run.scheduleOnce('sync', this, this.filterMovement);
+        run(() => this._nextScroll = run.scheduleOnce('sync', this, this.filterMovement));
```

Comment:
This isn't right, will stop the `scheduleOnce` behaviour from working. I added it to stop the "You have turned on testing mode..." autoRun error in my tests. But I assume we want the auto run (Or I guess you could use run.debounce? Not sure if that's more or less performant)

Will revert and try and fix my test another way.